public/video-ui/src/components/VideoPublishBar/VideoPublishBar.js

```diff
@@ -20,6 +21,7 @@ export default class VideoPublishBar extends React.Component {
 
   isPublishingDisabled() {
     return (
+      this.props.video.contentChangeDetails.scheduledLaunch ||
```

Comment:
I think this line is breaking this check. I saw this branch on code and I was never able to publish the video.